lib/scoped_search/auto_complete_builder.rb

```diff
@@ -208,7 +208,7 @@ def complete_value
         .uniq
         .map(&field.field)
         .compact
-        .map { |v| v.to_s =~ /\s/ ? "\"#{v}\"" : v }
+        .map { |v| v.to_s =~ /\s/ ? "\"#{v.gsub('"', '\"')}\"" : v }
```

Comment:
I wonder if there is no escape/quote method we can call, instead of doing the escaping manually?